prime-router/src/test/kotlin/fhirengine/utils/FhirTranscoderTests.kt

```diff
@@ -73,7 +73,7 @@ class FhirTranscoderTests {
         val decodedBundle = FhirTranscoder.decode(encodedBundle)
         // Ensure the decoded output is the same as the input pre-encode
         assertThat(decodedBundle).isNotNull()
-        assertThat(decodedBundle).equals(testBundle)
+        assertThat(decodedBundle == testBundle)
```

Comment:
I know the ticket said to use `.isEqualTo` but one of the perks of Kotlin is that to compare objects we can use == and skip creating `.equals` methods.

Reply:
Change to .isEqualsTo. That brings a lot more as it will print out proper error messages too.  == is the same as .equals. So that code is the same as `decodeBundle.equals(testBundle)`. If you use .isEqualsTo then it should perform that operation plus do some other printouts if it fails. If that really does not work then consider testing some of the properties for values

Reply:
.isEqualTo() compares that the objects are the same ex <...hir.r4.model.Bundle@[246ba501]> 
 = <...hir.r4.model.Bundle@[3f3134d4]>. I can change it to test the property values.